docs/docs/specs/v0.2/vault.md

```diff
@@ -0,0 +1,18 @@
+# Vault Application
+
+The HALO Vault is a Progressive Web App which stores and manages the user's HALO 0identity and ECHO data. It's most similar to a "wallet" or a "password manager".
+
+The HALO Vault also provides the DXOS Shell for use when logging into an application, and managing access to ECHO spaces. Shell UI is designed to be consumed via an iframe owned by the DXOS Client SDK.
+
+## Glossary
+- user (end-user): a user of a DXOS application
+- developer: builder of the DXOS application
+- identity: a representation of an actor or entity which can be authenticated by HALO
+- profile: a list of properties found on an identity object containing their medatada such as display name, avatar, ..., etc.
+- device: a particular instance of the dxos client (a browser / profile)
+
+## Scenarios
```

Comment:
Do we want to lay out future scenarios for this as well?

Reply:
perhaps in another iteration after we arrive at files? I think it's healthy for us to distill just the basics of drag and drop for now, and imply the need for cross-plugin node and action decoration. These scenarios (5, 6) will be plenty to go dogfooding and validating with people.

Reply:
there's only 3 here? I thought this was describing vault scenarios. For future scenarios I was imagining things like:
- discover apps/plugins
- open a space with a supported app based on compatibility with content within the space
- manage access of apps to vault
- etc.